Assets/Scripts/Town/PlayerInteractions.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class PlayerInteractions : MonoBehaviour
+{
+    private float interactionRadius = 3.0f;
```

Comment:
Move settings to configuration files, so they can be accessed using `DevSet`. Currently there is only one config called `ApplicationConfig`, which provides variables used for global application settings, so creating new settings set such as `TownSettings` and putting all configuration related to Town map would be good idea.